spec/rspec/core/configuration_spec.rb

```diff
@@ -1793,7 +1793,7 @@ def metadata_hash(*args)
         expect(config.full_backtrace?).to eq true
       end
 
-      it 'returns false when backtrace patterns isnt empty' do
+      it "returns false when backtrace patterns aren't empty" do
```

Comment:
```suggestion
      it "returns false when backtrace patterns are not empty" do
```
Contractions tend to be unclear for non-native english speakers, whilst you are here do you mind removing it?

Reply:
Good point, I didn't know that. Thank you :)